docs/hub/billing.md

```diff
@@ -89,5 +91,6 @@ View the full list of benefits at https://huggingface.co/subscribe/pro
 Similarly to the Enterprise Hub subscription, PRO subscriptions are billed like a typical subscription. The subscription renews automatically for you. You can choose to cancel the subscription at anytime in your billing settings: https://huggingface.co/settings/billing
 
 You can only pay for the PRO subscription with a credit card. The subscription is billed separately from any pay-as-you-go compute usage.
+Private repository storage above the [included storage](./storage-limits.md) will be billed along with your subscription renewal.
 
 Note: PRO benefits are also included in the Enterprise Hub subscription.
```

Comment:
```suggestion
Note: PRO benefits are also included in the Enterprise Hub subscription.

## Pay-as-you-go private storage 

Above the included 1TB (or 1TB per seat) of private storage in PRO and Enterprise Hub, private storage is invoiced at **$25/TB/month**, in 1TB increments.
```

For sake of completeness to add a line about PAYG storage here. I'm assuming this would apply for Individual users who just want to buy private storage.